src/apps/explorer/components/OrdersTableWidget/PaginationOrdersTable.tsx

```diff
@@ -119,10 +119,7 @@ const PaginationOrdersTable: React.FC = () => {
 
     return `${startPageCount} - ${endPageCount}`
   }
-  const hasPreviousPage = (): boolean => {
-    if (isLoading) return false
-    return pageOffset > 0 ? true : false
-  }
+  const hasPreviousPage = !isLoading && Boolean(pageOffset > 0)
```

Comment:
Sorry, another nitpick hehe

Comparison already returns a boolean :)

```suggestion
  const hasPreviousPage = !isLoading && pageOffset > 0
```

Reply:
Done, thanks for you feedback!